pycronofy/client.py

```diff
@@ -589,6 +590,24 @@ def authorize_with_service_account(self, email, scope, callback_url, state=None)
             endpoint="service_account_authorizations", data=params)
         None
 
+    def authorize_batch_with_service_account(self, service_account_authorizations, state=None):
```

Comment:
I think this could be confused with the batch endpoint, rather than being used to authorize multiple accounts.
Maybe something like this would be clearer?:
```suggestion
    def authorize_multiple_with_service_account(self, service_account_authorizations, state=None):
```

Reply:
Agreed. I have applied the naming suggestion throughout.